docs/ako_tenancy.md

```diff
@@ -66,3 +70,12 @@ metadata:
 ```
 
 With the above settings AKO will map the `n1` namespace to the `billing` tenant and all the objects will be created in that tenant .
+
+## Troubleshooting
+
+**Q: I've applied the `ako-tenant.json` role, but the controller UUID is not being populated. What should I do?**
+
+**A:** You need to assign the `ako-all-tenants-permission-controller.json` role to the "All Tenants" section in the Avi UI. This grants the AKO user the necessary read privileges for the controller across all tenants.
+
+To do this, navigate to `Administration -> Accounts -> Users` in the Avi UI, select the `ako-all-tenants-permission-controller` role, and ensure it's applied to the "All Tenants" section as shown below:
```

Comment:
Can we say `select the user after navigating to `Accounts-->Users` and then apply role in `Roles for all tenant section`?`